### ISSUE TITLE
is LRUCache unused ?

### ISSUE THREAD

**Issue by sathishk** [ssilverman/snowy-json#5]
We have class https://github.com/ssilverman/snowy-json/blob/master/src/main/java/com/qindesign/json/schema/util/LRUCache.java

This seems not referred to by any other class.

**Comment by ssilverman** [comment]
@sathishk I decided to leave this in because it's useful and because I'm still considering possibly using it. Can you tell me more about your requirements? For example, are you trying to build this project and squeeze out every last redundant byte? There are tools that exist that can remove redundant code from a build if size is critical for your application.

There are some unused methods, too, for instance, that aren't currently being used, but I can see as being useful for pluggable keywords and things like that. i.e. necessary for the API.

**Comment by sathishk** [comment]
Makes sense @ssilverman.

**Comment by ssilverman** [comment]
Have you tried running the

**Comment by ssilverman** [comment]
@sathishk My strategy for this library, before getting to v1.0.0, is to have it used in many different contexts. After feedback and more testing, then I'll optimize. Curious, have you tried the validator itself? I'd love your opinion and thoughts/experiences on its completeness and accuracy.

**Comment by sathishk** [comment]
@ssilverman agree with you. we should get the features before thinking about optimizing it.

I used the JSON schema validators sometime before (Draft 07). Recently I checked that snowy supports the new draft 2019/09. I just started experimenting with this. 

code looks great. Simple, Clean, and easy to understand. 

One suggestion is to include test cases. if you are interested I can contribute a few test cases.
